packages/block-library/src/link-preview/save.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+export default function save( { attributes } ) {
+	const { url, title, image, icon } = attributes;
+
+	if ( ! url ) {
+		return null;
+	}
+
+	return (
+		<a
+			{ ...useBlockProps.save( {
+				href: url,
+				className: image ? 'has-image' : undefined,
+			} ) }
+		>
+			{ image && <img src={ image } alt={ title } /> }
+			<div>
+				{ title && <strong>{ title }</strong> }
+				{ icon && (
+					<img
+						className="link-preview__icon"
+						src={ icon }
+						alt={ new URL( url ).host }
+					/>
+				) }
+				{ title ? new URL( url ).host.replace( /^www\./, '' ) : url }
+			</div>
```

Comment:
Same comment here as for `edit` - let's provide fallbacks if data is missing.

Reply:
Also the code here could be shared with the `edit` file.

Reply:
I already added a fallback: the fallback is simply the URL.